translation/transport_utils.go

```diff
@@ -107,18 +107,29 @@ func getCommandForParams(params []setParam) (command string) {
 /* Other transport-level helper functions */
 
 // wrp merges different values from a WDMP request into a WRP message
-func wrap(WDMP []byte, tid string, pathVars map[string]string, partnerIDs []string) (*wrp.Message, error) {
+func wrap(WDMP []byte, tid string, pathVars map[string]string, partnerIDs []string, traceHeaders []string) (*wrp.Message, error) {
 	canonicalDeviceID, err := wrp.ParseDeviceID(pathVars["deviceid"])
 	if err != nil {
 		return nil, transaction.NewBadRequestError(err)
 	}
 
+	if traceHeaders == nil {
```

Comment:
If `traceHeaders` is `nil` isn't that the same thing as the default value that `Headers` will be initiated to?

If so, I think this code block can be removed.

Reply:
You are correct. Thanks for catching that. Updated.